python/amici/petab_objective.py

```diff
@@ -0,0 +1,236 @@
+"""Functionality related to evaluating the objective function for a PEtab
+problem"""
+
+import numbers
+import numpy as np
+import pandas as pd
+import copy
+
+from typing import List, Sequence
+import amici
+import petab
+
+
+def edatas_from_petab(
+        model: amici.Model, measurement_df: pd.DataFrame,
```

Comment:
the construction process of edatas is also used to create simulation files from the rdatas. thus, this function should probably also be moved here to keep them in one place.

Reply:
sorry, missed it when reading the first time. copied already.